src/tests/Dictionary/DictionaryModal.test.jsx

```diff
@@ -60,11 +69,11 @@ describe('Test suite for dictionary modal', () => {
       ...props,
       organizations: [organizations],
     };
-    const wrap = shallow(<DictionaryModal {...newProps} />);
-    wrap
+    wrapper = shallow(<DictionaryModal {...newProps} />);
```

Comment:
Same as above.

Reply:
I had refactored the structure of the tests to align with the coding convention